test/complexity/measures/statistical_complexity.jl

```diff
@@ -0,0 +1,40 @@
+using ComplexityMeasures, Distances
+using Test
+
+@testset "Statistical Complexity" begin
+
+    m, τ = 6, 1
+    x = randn(10000)
+    c = StatisticalComplexity(
+        dist=JSDivergence(),
+        est=SymbolicPermutation(; m, τ),
+        entr=Renyi()
+    )
+    # complexity of white noise should be very close to zero
+    compl = complexity(c, x)
+    @test compl isa Real
+    @test 0 < compl < 0.02
+    # complexity of monotonically ascending signal should be exactly zero
+    compl = complexity(c, collect(1:100))
+    @test compl == 0.0
+
```

Comment:
Shouldn't we have a test here for something that should have "high" complexity?

Reply:
Yes, but what here would be the criterium for "high" complexity? Close (how close) to the maximum curve? Since I cannot reproduce the results from the Rosso paper, and am by now pretty sure it's not me who made a mistake, I don't really know any dynamical system that would produce such a high complexity. 

We could of course test that the complexity value is between the minimum and maximum curve, which we test separately. I'll add that test

Reply:
okay, what I meant is to test something that has medium entropy. at the moment we have entropy 1 and entropy 0 cases.